src/v/cluster/topic_updates_dispatcher.cc

```diff
@@ -102,6 +129,27 @@ topic_updates_dispatcher::apply_update(model::record_batch b) {
       });
 }
 
+ss::future<> topic_updates_dispatcher::update_leaders_with_estimates(
+  std::vector<ntp_leader> leaders) {
+    for (const auto& i : leaders) {
+        vlog(
+          clusterlog.info,
```

Comment:
debug severity maybe ? I am thinking about topics with thousands of partitions

Reply:
Good catch, several noisy log message in here were committed by mistake, will remove.